packages/lang/llvm/package.mk

```diff
@@ -28,11 +27,10 @@ PKG_CMAKE_OPTS_COMMON="-DLLVM_INCLUDE_TOOLS=ON \
                        -DLLVM_ENABLE_SPHINX=OFF \
                        -DLLVM_ENABLE_OCAMLDOC=OFF \
                        -DLLVM_ENABLE_BINDINGS=OFF \
-                       -DLLVM_TARGETS_TO_BUILD=AMDGPU \
                        -DLLVM_ENABLE_TERMINFO=OFF \
                        -DLLVM_ENABLE_ASSERTIONS=OFF \
                        -DLLVM_ENABLE_WERROR=OFF \
-                       -DLLVM_ENABLE_ZLIB=ON \
+                       -DLLVM_ENABLE_ZLIB=OFF \
```

Comment:
is this change needed to make it work or just some cleanup ?

Reply:
It was required to get rust to link properly otherwise it would complain about missing zlib symbols. I didn't have time to look into it and figure zlib support probably wasn't needed anyways.